src/main/scala/diplomacy/BundleBridge.scala

```diff
@@ -122,18 +122,24 @@ class BundleBridgeNexus[T <: Data](
 }
 
 object BundleBridgeNexus {
+  def safeRegNext[T <: Data](x: T): T = {
```

Comment:
should this be moved to rocket-chip Utils?

Reply:
The fact that I needed this is due to a detail of the BundleBroadcast. I'd rather not promote an alternative RegNext, especially, since it might get fixed in the future.

Reply:
ok

Reply:
Yeah, I agree with @terpstra here. Thanks for your trepidation about promoting an alternative `RegNext`. Trying to keep Rocket Chip-dialect Chisel close to upstream Chisel is a noble goal. There does need to be an upstream solution to this.